app/crawler/crawler.py

```diff
@@ -51,26 +30,32 @@ class Crawler(object):
     def __init__(self, queries,
                  fetcher=PhantomFetcher(),
                  archiver=Archiver(),
-                 queuer=Queuer(),
+                 queuer=None,
                  version=1,
-                 download=False,
-                 wait_query=None):
+                 archive=False,
+                 wait_query=None,
+                 priority=0):
         self.queries = queries
         self.version = version
-        self.download = download
+        self.archive = archive
         self.wait_query = wait_query
         self.fetcher = fetcher
         self.archiver = archiver
         self.queuer = queuer
+        self.priority = priority
 
     def crawl(self, url):
+        print("fetching %s" % url)
         html = self.fetcher.fetch(url, self.wait_query)
```

Comment:
Fetching is just getting the page. Crawling is moving from page to page. 

To be honest, that isn't exactly what the crawler class does on its own, but this is the closest I got to having a concise description